src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -43,18 +43,13 @@ export default class VivViewerLayer extends CompositeLayer {
       dtype
     });
     const getTileData = async ({ x, y, z }) => {
-      const { data, width, height } = await loader.getTile({
+      const tile = await loader.getTile({
         x,
         y,
         z: -z,
         loaderSelection
       });
-      if (width !== tileSize || height !== tileSize) {
-        throw Error(
-          `Tile data  { width: ${width}, height: ${height} } does not match tilesize: ${tileSize}`
-        );
-      }
-      return data;
```

Comment:
Is  `tileSize` not a prop anymore? If it still is something passed to the layer, then it would be useful to have something that checks to make sure the data returned in the callback is the right size. If `loader.getTile` were to return something with variable size, would that still work?

Reply:
Yes, that is actually the idea here - not all tiles are equally sized in the `bioformats` pyramid and the metadata surrounding it also appears to be incorrect.  It doesn't have an affect now since the metadata is wrong, but we should allow this IMO and not enforce checking - perhaps a warning....